src/memray/commands/attach.py

```diff
@@ -304,6 +415,32 @@ def prepare_parser(self, parser: argparse.ArgumentParser) -> None:
 
     def run(self, args: argparse.Namespace, parser: argparse.ArgumentParser) -> None:
         verbose = args.verbose
+        mode: TrackingMode = "ACTIVATE"
+        duration = None
+        heap_size = None
+
+        if args.stop_tracking:
+            if args.output:
+                parser.error("Can't use --stop-tracking with -o or --output")
+            if args.force:
+                parser.error("Can't use --stop-tracking with -f or --force")
+            if args.aggregate:
+                parser.error("Can't use --stop-tracking with --aggregate")
+            if args.native:
+                parser.error("Can't use --stop-tracking with --native")
+            if args.follow_fork:
+                parser.error("Can't use --stop-tracking with --follow-fork")
+            if args.trace_python_allocators:
+                parser.error("Can't use --stop-tracking with --trace-python-allocators")
```

Comment:
Maybe

```python
incompatible_flags = {
    args.output: "--output",
    args.force: "--force",
    args.aggregate: "--aggregate",
    args.native: "--native",
    args.follow_fork: "--follow-fork",
    args.trace_python_allocators: "--trace-python-allocators",
    args.no_compress: "--no-compress",
}

if args.stop_tracking and any(incompatible_flags):
    conflicting_flags = [message for flag, message in incompatible_flags.items() if flag]
    parser.error(f"Can't use --stop-tracking with {', '.join(conflicting_flags)}")
```

Reply:
@godlygeek what do you prefer?

Reply:
Hm. Well, as it's implemented right now, we don't know whether `-o` or `--output` was used, but we print an error message that says "Can't use --stop-tracking with -o or --output".

If we went with your proposed approach, we'd get "Can't use --stop-tracking with --output" which I worry might confuse people if they used `-o` instead of `--output`.

I dunno. It's less verbose, but I'm not convinced it's better - and it's about the same amount of maintenance, I think.

Reply:
Ok, let's leave as it is 👍